<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionDfs"/>
    <addaction name="actionBfs"/>
    <addaction name="actionBest_first"/>
    <addaction name="actionPause"/>
   </widget>
   <addaction name="menuAction"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDfs">
   <property name="text">
    <string>dfs</string>
   </property>
  </action>
  <action name="actionBfs">
   <property name="text">
    <string>bfs</string>
   </property>
  </action>
  <action name="actionBest_first">
   <property name="text">
    <string>best-first</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>reset</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>stop</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
